mail_as_letter/static/description/icon.svg

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
so my knowledge, odoo searches for .png or .jpg files in this folder, and my assumption was that you accidentally git added the .svg file instead of the png